operations/observability/mixins/meta/dashboards/components/ws-manager-bridge.json

```diff
@@ -392,77 +418,61 @@
         "x": 12,
         "y": 9
       },
-      "id": 54,
+      "id": 60,
       "options": {
         "legend": {
           "calcs": [],
           "displayMode": "list",
-          "placement": "bottom"
+          "placement": "bottom",
+          "showLegend": true
         },
         "tooltip": {
           "mode": "single",
           "sort": "none"
         }
       },
       "targets": [
-        {
-          "datasource": {
-            "type": "prometheus",
-            "uid": "P4169E866C3094E38"
-          },
-          "editorMode": "code",
-          "expr": "histogram_quantile(0.5, sum(rate(gitpod_ws_manager_bridge_workspace_instance_update_completed_seconds_bucket{db_write=\"true\", cluster=~\"$cluster\", pod=~\"$pod\"}[5m])) by (le))",
-          "legendFormat": "50th",
-          "range": true,
-          "refId": "A"
-        },
         {
           "datasource": {
             "type": "prometheus",
             "uid": "${datasource}"
           },
           "editorMode": "code",
-          "expr": "histogram_quantile(0.90, sum(rate(gitpod_ws_manager_bridge_workspace_instance_update_completed_seconds_bucket{db_write=\"true\", cluster=~\"$cluster\", pod=~\"$pod\"}[5m])) by (le))",
-          "hide": false,
-          "legendFormat": "90th",
+          "expr": "sum(increase(gitpod_ws_instances_marked_stopped_total[5m])) by (previous_phase)",
+          "legendFormat": "__auto",
           "range": true,
-          "refId": "B"
-        },
-        {
-          "datasource": {
-            "type": "prometheus",
-            "uid": "${datasource}"
-          },
-          "editorMode": "code",
-          "expr": "histogram_quantile(0.99, sum(rate(gitpod_ws_manager_bridge_workspace_instance_update_completed_seconds_bucket{db_write=\"true\", cluster=~\"$cluster\", pod=~\"$pod\"}[5m])) by (le))",
-          "hide": false,
-          "legendFormat": "99th",
-          "range": true,
-          "refId": "C"
+          "refId": "A"
         }
       ],
-      "title": "Processing latency by write replicas [1m] (bucketed)",
+      "title": "Instances marked as stopped",
```

Comment:
I'd recommended adding `[5m]` at the end of the graph title to indicate what's the aggregation period